src/layer/marker/Marker.Drag.js

```diff
@@ -60,6 +60,8 @@ L.Handler.MarkerDrag = L.Handler.extend({
 
 		// @event movestart: Event
 		// Fired when the marker starts moving (because of dragging).
+
+		this._marker._oldLatLng = this._marker.getLatLng();
```

Comment:
Yeah, I went back and forth on this quickly for initial commit. You're right. New commit added.